server/backends/chunkstore/chunkstore.go

```diff
@@ -88,24 +89,39 @@ func (c *Chunkstore) DeleteBlob(ctx context.Context, blobName string) error {
 	}
 }
 
-func chunkName(blobName string, index uint16) string {
+func ChunkName(blobName string, index uint16) string {
 	return blobName + "_" + fmt.Sprintf("%04x", index)
 }
 
-func (c *Chunkstore) chunkExists(ctx context.Context, blobName string, index uint16) (bool, error) {
-	return c.internalBlobstore.BlobExists(ctx, chunkName(blobName, index))
+func (c *Chunkstore) GetLastChunkIndex(ctx context.Context, blobName string) (uint16, error) {
```

Comment:
Is this slow? Seems like a lot of round trips. Might be something we can optimize away later by saving the last chunk index in the invocation row when we Finalize it

Reply:
that makes sense; this definitely could be slow depending on the number of blobs / latency to the blobstore

Reply:
Leveraging the invocation table for this now.